### ISSUE TITLE
Copy with ctrl+c only when highlighted

### ISSUE THREAD

**Issue by jacobtomlinson** [zeit/hyper#3798]
- [ ] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [ ] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: Windows 10
- **Hyper.app version**: 3.0.2
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: Default
- **Relevant information from devtools** _(CMD+ALT+I on macOS, CTRL+SHIFT+I elsewhere)_: NA
- **The issue is reproducible in vanilla Hyper.app**: `Is Vanilla`

## Issue
If I highlight some text and hit ctrl+c I would like to copy the text to the clipboard. If no text is highlighted I would like ctrl+c to interrupt as normal.

This seems to be the way the terminal works in VSCode and I am trying to make Hyper consistent with that.

Is it possible to configure this functionality?

**Comment by actual-size** [comment]
I agree with @jacobtomlinson, Windows Terminal also behaves this way and Hyper should definitely follow the standard they're setting
https://github.com/microsoft/terminal/issues/839#issuecomment-492892232

**Comment by horizone** [comment]
Oh yes, the possibility to use Ctrl-C for copy, only when something is highlighted would be what a user expects. There is no real risk for accidentally stopping a program because one rarely tries to copy something as programs are running in the terminal anyway, only when the prompt is ready to have something pasted into, you tend to copy and paste stuff.
